v21.2/import.md

```diff
@@ -22,7 +22,7 @@ The `IMPORT` [statement](sql-statements.html) imports the following types of dat
 - `IMPORT` cannot be used with [user-defined types](create-type.html). Use [`IMPORT INTO`](import-into.html) instead.
 - `IMPORT` is a blocking statement. To run an import job asynchronously, use the [`DETACHED`](#options-detached) option.
 - `IMPORT` cannot be used within a [rolling upgrade](upgrade-cockroach-version.html).
-- {% include {{page.version.version}}/sql/import-into-regional-by-row-table.md %}
+- `IMPORT` cannot directly import data to `REGIONAL BY ROW` tables that are part of [multi-region databases](multiregion-overview.html). Instead, use [`IMPORT INTO`](import-into.html) which now supports importing into `REGIONAL BY ROW` tables.
```

Comment:
suggest removing the already implied "now" (it's always now on the internet)

if you want to call out the timeliness aspect of this being a new thing in v21.2, maybe an edit like

"IMPORT cannot directly .... new in v21.2: Instead, use IMPORT INTO which supports importing into REGIONAL BY ROW tables"

also, FWIW as an outsider I find the IMPORT vs IMPORT INTO distinction quite confusing - esp if feature sets are different. nothing to do from the POV of this PR however ....